### PR TITLE
Add comparison and logical operators

### PR DESCRIPTION
**Pull request by romasku**
This should make https://github.com/neuromation/neuro-flow/issues/50 more useful.

### CODE REVIEW

File: neuro_flow/cli.py [hunk 1]
```diff
@@ -308,3 +308,7 @@ async def show(
             BatchRunner(config_dir, client, storage)
         )
         await runner.logs(bake_id, task_id, attempt_no=attempt, raw=raw)
+
+
+if __name__ == "__main__":
```
Comment (romasku) [inline neuro_flow/cli.py:313]: This allows to run our tool using `python -m neuro_flow.cli args_here`. This allows me to debug code in Pycharm. Should I revert this changes? Maybe I'm missing some more elegant way to debug the code.

Comment (asvetlov) [reply neuro_flow/cli.py:313]: I'm fine with it if it makes your life easier.
I don't use pycharm but debug directly with pdb++

File: neuro_flow/expr.py [hunk 2]
```diff
@@ -313,6 +314,37 @@ def make_text(arg: Token) -> Text:
     return Text(arg.start, arg.end, arg.value)
 
 
+@dataclasses.dataclass(frozen=True)
+class BinOp(Item):
+    op: Callable[[TypeT, TypeT], TypeT]
+    args: Tuple[Item, Item]
```
Comment (asvetlov) [inline neuro_flow/expr.py:320]: Please use explicit *left* and *right* operands instead of 2-items tuple.
This is `BinOp` class, the number of arguments is always 2.

**Review by asvetlov — CHANGES_REQUESTED** [review]
please support unary operator `not` for completeness

Side note: maybe we will need to implement arithmetic operators in the near future, who knows?

**Review by asvetlov — APPROVED** [review]
